src/ibex_bluesky_core/devices/dae/dae_controls.py

```diff
@@ -43,4 +43,4 @@ def __init__(self, dae_prefix: str, name: str = "") -> None:
     @AsyncStatus.wrap
     async def set(self, value: BeginRunExBits) -> None:
```

Comment:
The parent class (`Movable`) version of the `set` function returns a type Status (from `bluesky.protocols`), but this one returns `None`.